### PR TITLE
refactor(autoware_pointcloud_preprocessor): rework concatenate_pointcloud and time_synchronizer_node parameters

### PR DESCRIPTION
**Pull request by vividf**
## Description
This PR includes the following changes
1. Remove the default parameters for the time synchronizer node and concatenate pointcloud. 
2. Add parameter, launch file, and parameters schema for the nodes. 
3. Change the name `nodelet` to `node`.

Note that I didn't update the readme since after this PR https://github.com/autowarefoundation/autoware.universe/pull/8300 the readme file will change a lot (parameters are different). 
I will update the readme in this PR https://github.com/autowarefoundation/autoware.universe/pull/8300

## Related links

**Parent Issue:**

- Link

## How was this PR tested?
`ros2 launch autoware_pointcloud_preprocessor concatenate_pointcloud.launch.xml`
and 
`ros2 launch autoware_pointcloud_preprocessor time_synchronizer_node.launch.xml`
## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

**Review by yukkysaito — APPROVED** [review]

File: sensing/autoware_pointcloud_preprocessor/schema/concatenate_pointclouds.schema.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Concatenate Pointclouds Node",
+  "type": "object",
+  "definitions": {
+    "concatenate_pointclouds": {
+      "type": "object",
+      "properties": {
+        "timeout_sec": {
+          "type": "number",
+          "default": 0.1,
```
Comment (amadeuszsz) [inline sensing/autoware_pointcloud_preprocessor/schema/concatenate_pointclouds.schema.json:11]: Assign default values as string

File: sensing/autoware_pointcloud_preprocessor/schema/concatenate_pointclouds.schema.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "default": [],
+          "description": "List of input topics."
+        },
+        "max_queue_size": {
+          "type": "integer",
+          "default": 5,
```
Comment (amadeuszsz) [inline sensing/autoware_pointcloud_preprocessor/schema/concatenate_pointclouds.schema.json:37]: Assign default values as string

File: sensing/autoware_pointcloud_preprocessor/schema/time_synchronizer_node.schema.json [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Time synchronizer Node",
+  "type": "object",
+  "definitions": {
+    "time_synchronizer": {
+      "type": "object",
+      "properties": {
+        "timeout_sec": {
+          "type": "number",
+          "default": 0.033,
```
Comment (amadeuszsz) [inline sensing/autoware_pointcloud_preprocessor/schema/time_synchronizer_node.schema.json:11]: Assign default values as string

File: sensing/autoware_pointcloud_preprocessor/schema/time_synchronizer_node.schema.json [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+          "description": "List of input topics."
+        },
+        "synchronized_pointcloud_postfix": {
+          "type": "string",
+          "default": "pointcloud",
+          "description": "Postfix for the synchronized point cloud."
+        },
+        "max_queue_size": {
+          "type": "integer",
+          "default": 5,
```
Comment (amadeuszsz) [inline sensing/autoware_pointcloud_preprocessor/schema/time_synchronizer_node.schema.json:53]: Assign default values as string

**Review by amadeuszsz — CHANGES_REQUESTED** [review]
Apart of comments, please consider adding boundaries for all integer / numeric parameters.

File: sensing/autoware_pointcloud_preprocessor/schema/concatenate_pointclouds.schema.json (hunk 1 above)
Comment (vividf) [reply sensing/autoware_pointcloud_preprocessor/schema/concatenate_pointclouds.schema.json]: Thanks! fixed them in https://github.com/autowarefoundation/autoware.universe/pull/8509/commits/08ec2189e062a6b8e4cb21748c703469a4120e0c

**Review by amadeuszsz — APPROVED** [review]
LGTM

**Pull request merged**